cfme/networks/views.py

```diff
@@ -646,6 +646,30 @@ def is_displayed(self):
                 self.title.text == '{name} (Summary)'.format(name=self.context['object'].name))
 
 
+class ParentWithSubnetView(BaseLoggedInPage):
```

Comment:
Consider `NetworkEntitySubnetView` to be a bit more descriptive here?

Please update the docblock to be more complete - this is a view for use with a network entity that includes `SubnetCollection` in its `_collections`

Reply:
Thanks for the name suggestion, I knew it was kinda bad :), but I didn't have any better ideas. I included this changes in new push.